Packs/FeedCognyteLuminar/Integrations/CognyteLuminar/CognyteLuminar.yml

```diff
@@ -0,0 +1,149 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Luminar_IOCs_and_leaked_credentials
+  version: -1
+configuration:
+- additionalinfo: Luminar Base URL
+  defaultvalue: https://www.cyberluminar.com
+  display: Luminar Base URL
+  name: luminar_base_url
+  required: true
+  type: 0
+- additionalinfo: Luminar API Account ID
+  display: Luminar API Account ID
+  name: luminar_account_id
+  required: true
+  type: 0
+- additionalinfo: Luminar API Client ID
+  display: Luminar API Client ID
+  name: luminar_client_id
+  required: true
+  type: 0
+- additionalinfo: Luminar API Secret
+  display: Luminar API Client Secret
+  name: luminar_client_secret
+  required: true
+  type: 4
+- additionalinfo: Trust any certificate (not secure)
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- additionalinfo: Use system proxy settings
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'true'
+  display: Fetch indicators
+  type: 8
+  required: false
+  name: feed
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation.
+  defaultvalue: feedInstanceReputationNotSet
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: F - Reliability cannot be judged
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ''
+  type: 17
+  required: false
+  options:
+    - never
+    - interval
+    - indicatorType
+    - suddenDeath
+  name: feedExpirationPolicy
+- display: ''
+  type: 1
+  required: false
+  name: feedExpirationInterval
+- display: Feed Fetch Interval
+  type: 19
+  required: false
+  name: feedFetchInterval
+- display: Tags
+  required: false
+  type: 0
+  additionalinfo: Supports CSV values.
+  name: feedTags
+- display: Traffic Light Protocol Color
+  options:
+    - RED
+    - AMBER
+    - GREEN
+    - WHITE
+  required: false
+  type: 15
+  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
+  name: tlp_color
```

Comment:
doesn't seem like this integration parameter is handles in the code. Please add this as one of the fields to the indicator, like this:
        if tlp_color:
            indicator['fields']['trafficlightprotocol'] = tlp_color